README.md

```diff
@@ -76,21 +113,37 @@ You can observe that:
   kubectl get pods -n liqo-demo -o wide  
 ``` 
 
-To get more information about how to install Liqo in your own clusters and configure it, you can check out the [Liqo Documentation](https://doc.liqo.io/user/).
+## Installation
+
+Liqo can be installed by using Helm. The configuration depends on the cluster type (K8s, K3s, managed) and the provider. Possible scenarios can be found [here](https://doc.liqo.io/user/scenarios/).
 
-## Architecture
+To be completed
 
-Liqo relies on several components:
+## Roadmap
```

Comment:
There's a file ROADMAP.md in the repo.
Let's keep it, providing a link here to the file.
Do not duplicate content.